docs/payid-discovery.md

```diff
@@ -0,0 +1,46 @@
+---
+id: payid-discovery
+title: Discover a PayID
+sidebar_label: Discover a PayID
+---
+
+The PayID Discovery endpoint facilitates finding the PayID URI for a given PayID.
```

Comment:
Would it make sense to link to the release notes? Not sure if versions matter much for this.

Reply:
I will in the future, but in this case, this has been around a while.